sdk/communication/azure-communication-administration/azure/communication/administration/_phone_number_administration_client.py

```diff
@@ -80,520 +45,139 @@ def from_connection_string(
+        "geographic".
+        :type number_type: str or ~azure.communication.administration.models.PhoneNumberType
+        :keyword assignment_type: Required. The phone number's assignment type. Possible values include:
+        "person", "application".
+        :type assignment_type: str or ~azure.communication.administration.models.AssignmentType
+        :keyword capabilities: Required. The phone number's capabilities.
+        :type capabilities: ~azure.communication.administration.models.SearchCapabilities
+        :keyword area_code: The desired area code.
+        :type area_code: str
+        :keyword quantity: The desired quantity of phone numbers.
+        :type quantity: int
+        :rtype: ~azure.core.polling.LROPoller[~azure.communication.administration.SearchResult]
+        """
+        pass
+
+    @distributed_trace
+    def begin_purchase_phone_numbers(
         self,
-        pstn_configuration,  # type: PstnConfiguration
-        phone_number,  # type: str
-        **kwargs  # type: Any
+        search_id, #type: str
+        **kwargs # type: Any
     ):
-        # type: (...) -> None
-        """Endpoint for configuring a pstn number.
-
-        :param pstn_configuration: Definition for pstn number configuration.
-        :type pstn_configuration: ~azure.communication.administration.PstnConfiguration
-        :param phone_number: The phone number to configure.
-        :type phone_number: str
-        :rtype: None
+        #type: (str, Any) -> LROPoller[SearchResult]
+        """Begins purchasing a phone number search
+        :param str search_id: Id for the search
+        :rtype: ~azure.core.polling.LROPoller[~azure.communication.administration.SearchResult]
         """
-        return self._phone_number_administration_client.phone_number_administration.configure_number(
-            pstn_configuration,
-            phone_number,
-            **kwargs
-        )
+        pass
 
     @distributed_trace
-    def unconfigure_number(
+    def get_search_result(
         self,
-        phone_number,  # type: str
-        **kwargs  # type: Any
+        search_id, #type: str
+        **kwargs # type: Any
     ):
-        # type: (...) -> None
-        """Endpoint for unconfiguring a pstn number by removing the configuration.
-
-        :param phone_number: The phone number in the E.164 format.
-        :type phone_number: str
-        :rtype: None
+        #type: (str, Any) -> SearchResult
+        """Gets the search associated with the id
+        :param str search_id: Id for the search
+        :returns: Search result associated with the search id
+        :rtype: ~azure.communication.administration.SearchResult
         """
-        return self._phone_number_administration_client.phone_number_administration.unconfigure_number(
-            phone_number,
-            **kwargs
-        )
+        pass
 
     @distributed_trace
-    def list_phone_plan_groups(
+    def get_operation(
```

Comment:
We normally wouldn't expose this directly - rather it would be part of rehydrating a LROPoller.
Are other languages doing a generic `get_operation` API?